openpgp.d.ts

```diff
@@ -297,31 +298,33 @@ export class Message<T extends MaybeStream<Data>> {
 
 /* ############## v5 CONFIG #################### */
 
-export namespace config {
-  let preferHashAlgorithm: enums.hash;
-  let encryptionCipher: enums.symmetric;
-  let compression: enums.compression;
-  let showVersion: boolean;
-  let showComment: boolean;
-  let integrityProtect: boolean;
-  let debug: boolean;
-  let deflateLevel: number;
-  let aeadProtect: boolean;
-  let ignoreMdcError: boolean;
-  let checksumRequired: boolean;
-  let rsaBlinding: boolean;
-  let minRsaBits: number;
-  let passwordCollisionCheck: boolean;
-  let revocationsExpire: boolean;
-  let useNative: boolean;
-  let zeroCopy: boolean;
-  let tolerant: boolean;
-  let versionString: string;
-  let commentString: string;
-  let keyserver: string;
-  let nodeStore: string;
-  let allowInsecureDecryptionWithSigningKeys: boolean;
-}
+interface Config {
+  preferHashAlgorithm: enums.hash;
+  encryptionCipher: enums.symmetric;
+  compression: enums.compression;
+  showVersion: boolean;
+  showComment: boolean;
+  integrityProtect: boolean;
+  deflateLevel: number;
+  aeadProtect: boolean;
+  ignoreMdcError: boolean;
+  checksumRequired: boolean;
+  minRsaBits: number;
+  passwordCollisionCheck: boolean;
+  revocationsExpire: boolean;
+  tolerant: boolean;
+  versionString: string;
+  commentString: string;
+  keyserver: string;
+  nodeStore: string;
+  allowInsecureDecryptionWithSigningKeys: boolean;
+  v5Keys: boolean;
+}
+export var config: Config;
```

Comment:
I think it'd make sense to rename `openpgp.config` to `openpgp.defaultConfig` given the changes.

Reply:
I think it might be confusing, as `openpgp.defaultConfig` would in turn also have a default :s And `openpgp.config` is still something that you can change, not just a description of the defaults, so I would keep it as is. I think it's pretty obvious that the configs passed to the functions are more specific and override these ones.